helpdesk_mgmt_project/models/project_task.py

```diff
@@ -0,0 +1,18 @@
+from odoo import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    ticket_count = fields.Integer(
+        compute='_compute_ticket_count',
+        string="Ticket Count")
```

Comment:
The same suggestion as above with projects

Reply:
Idem